pyproject.toml

```diff
@@ -45,7 +45,7 @@ click = ">=7.1.2,<9.0"  # NOTE: COLAB has 7.1.2 and has problems updating
 rich = "^10.15.2"
 shellingham = "^1.4.0"
 scikit-learn = ">=0.24.0"
-Shapely = ">=1.7.1"
+Shapely = "^1.8.1"
```

Comment:
@gatli didn't we run into problems with Shapely at some point?

Reply:
Yup, we can't use it. It doesn't have wide enough binary support.

Reply:
So what should we do about this? @Anirudh-Scale how much do we rely in the shapely dependency? Any chance we can get around this? If not, @gatli how can we still make this work?

Reply:
@pfmark I can implement a function to calculate the area intersection without Shapely, it's just going to be significantly slower. If that's a trade-off we're ok with, I can do that.

Reply:
There's already a function for polygon area intersection in the directory under `geometry.py`. I'd suggest using that since it looks like you're using shapely only for polygon intersection area.